_posts/2023-07-22-gsoc-sqlancer-final.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 'GSoC 2023: Final Report of test-case reduction'
+date: 2023-8-30
+author: Yutan Yang
+categories:
+  - blog
+tags:
+  - gsoc
+---
+
+## Overview
+
+SQLancer generates a large number of statements, but not all of them are relevant to the bug. To automatically reduce the test cases, [Yutan Yang](https://github.com/ColinYoungTaro) implemented two reducers in SQLancer, the statement reducer and the AST-based reducer.
```

Comment:
With the recent changes, your profile is now displayed next to your post, so perhaps you can write the post in first person (I), rather than third person (Yutan)?